index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Salmon Cookies!!</title>
+    <link href="css/style.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Cormorant+Garamond|Do+Hyeon" rel="stylesheet">
+  </head>
+  <body>
+    <header>
+        <h1>Pat's Salmon Cookies!</h1>
+        <h2>Trust me, you're gonna love 'em!</h2>
+    </header>
+    <img src="/img/salmon.png" alt="salmon">
```

Comment:
Watch out: with this leading slash, the image link doesn't work unless I'm using liveserver. Get rid of the leading slash to make sure this'll work when I just use `open index.html`.